src/server/routes/actions.ts

```diff
@@ -26,19 +33,86 @@ router.get<unknown, BalanceResponse>("/balance", (_, res) => {
     });
 });
 
+const missingParameterError = (
+  res: Response<DripErrorResponse>,
+  parameter: keyof BotDripRequestType | keyof FaucetRequestType,
+): void => {
+  res.status(400).send({ error: `Missing parameter: '${parameter}'` });
+};
+
+const addressMiddleware = (
+  req: Request<unknown, DripResponse, Partial<DripRequestType>>,
+  res: Response,
+  next: NextFunction,
+): void => {
+  if (!req.body.address) {
+    return missingParameterError(res, "address");
+  }
+  next();
+};
+
 const dripRequestHandler = new DripRequestHandler(actions, storage, recaptchaService);
 
+router.post<unknown, DripResponse, FaucetRequestType>("/faucet", addressMiddleware, async (req, res) => {
```

Comment:
nit: could we change a name to something which is not duplicating the project name and has some actionable meaning? some kind of `/drip/web` `/drip/bot` or something else which would make it more clear from the url